### ISSUE TITLE
Submission license agreement step not accessible

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1764]
The license texts on the submission license agreement step are not accessible to keyboard navigation or screenreaders. These text boxes should be focused with keyboard navigation and should be read by a screenreader. They should also be associated with the input-checkbox using the proper ARIA attribute.
